### PR TITLE
User Story 7 - Favorites

### PR DESCRIPTION
**Pull request by kevinldg**
Ability to mark upcycling ideas as favorite and view them on a seperate page (favorite page).

Corresponding user story: https://github.com/kevinldg/upcycling-app/issues/8

### CODE REVIEW

File: components/Card/index.js [hunk 1]
```diff
@@ -20,9 +24,43 @@ const Hashtags = styled.ul`
...
+  isFavourite,
+}) {
   return (
     <Article>
+      <FavoriteButton onClick={() => onToggleFavourites({ id })}>
+        {isFavourite ? (
+          <StyledIcon icon={solidHeart} />
+        ) : (
+          <StyledIcon icon={regularHeart} />
+        )}
```
Comment (doemser) [inline components/Card/index.js:62]:
```suggestion
     
          <StyledIcon icon={isFavourite ? solidHeart : regularHeart} />
    
```

File: pages/_app.js [hunk 2]
```diff
@@ -30,17 +32,29 @@ export default function App({ Component, pageProps }) {
     }
   }
 
+function handleToggleFavourites({ id }) {
```
Comment (doemser) [inline pages/_app.js:35]: I would expect that just the id is passed, here you destructure an object that contains the id key.

File: pages/index.js [hunk 3]
```diff
@@ -21,7 +21,21 @@ const CardListItem = styled.li`
...
+    text-decoration: underline;
+  }
+`;
+
+export default function HomePage({
+  ideas,
+  onToggleFavourites,
+  favouriteIdeas,
+}) {
+  console.log(favouriteIdeas);
```
Comment (doemser) [inline pages/index.js:38]: remove console.log

File: components/Card/index.js [hunk 4]
```diff
@@ -20,9 +24,43 @@ const Hashtags = styled.ul`
...
+  image,
+  title,
+  hashtags,
+  onToggleFavourites,
+  id,
+  isFavourite,
+}) {
   return (
     <Article>
+      <FavoriteButton onClick={() => onToggleFavourites({ id })}>
```
Comment (doemser) [inline components/Card/index.js:57]: just pass id here

**Review by doemser — CHANGES_REQUESTED** [review]
Good Work!

Just some minor changes requested.

**Comment by kevinldg** [comment]
> Good Work!
> 
> Just some minor changes requested.

Thx for your feedback @doemser ! We've implemented the changes as requested 👍

**Review by doemser — APPROVED** [review]

**Pull request merged**